serveur_calcul_python/tests/unit/test_gen_graphique.py

```
@@ -0,0 +1,358 @@
+Interpretation error checks
+Checks that errors are raised when invalid inputs are provided. 
+'''
+def test_raise_invalid_cubf_error():
+    '''
+    # test_raise_invalid_cubf
+    Tests whether the an error is raised whan a land use code is 
+    provided which is larger than the provided input
+    '''
+    with pytest.raises(ValueError) as excinfo:
+        input_dict=GGIDS.provide_invalid_cubf()
+        x_values_chart,regs,reg_sets,land_use,units,case = FCDP.parse_dict_for_graphs(input_dict)
```

Ruff (0.15.20)

[warning] `pytest.raises()` block should contain a single simple statement

(PT012)